.travis.yml

```diff
@@ -1,14 +1,12 @@
 language: python
 cache: pip
+dist: bionic
 env:
   global:
     - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
     - SEGFAULT_SIGNALS=all
 jobs:
   include:
-    - python: '3.6'
-      env:
-        - TOXENV=py36-notoml,report,check,docs
```

Comment:
In removing this we've lost the check and docs step, mind adding again? Probably worth adding to 3.10 rather than 3.7.

Reply:
Fixed